frontend/utils/giLodash.js

```diff
@@ -19,6 +19,12 @@ export function pick (o: Object, props: string[]): {...} {
   return x
 }
 
+export function choose (a: Array<*>, indicies: Array<number>): Array<*> {
+  const x = []
+  for (const idx of indicies) { x.push(a[idx]) }
+  return x
+}
+
```

Comment:
OK, changed `a` to `array`, and `indicies` to `indices`.

However, out-of-bounds, and negative indices I'm not going to handle because JS already handles those its own way. I don't want to add duplicate error-handling code because it's redundant and slows down performance.